contributing.md

```diff
@@ -0,0 +1,27 @@
+# How To Become A Contributor And Submit Your Own Code
+There are few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
+
+## Getting Started
+This repo uses Lerna(https://lernajs.io/) as a top level setup, which is a tool that optimizes the workflow around managing multi-package repositories with git and npm.
+
+* Make sure you have forked the desired repo(https://github.com/electrode-io/electrode.git) for developing and testing your code changes.
+
+* Make sure you have installed Lerna
+
+````
+$ npm install --global lerna
+$ lerna init
+````
+
+* Run `lerna bootstrap` for bootstrapping the packages in the current Lerna repo. It will install all their dependencies and linking any cross-dependencies.
+
+* Run `gulp samples-local` for pulling electrode packages from local under `/packages` folder.
+
+* Now you can go to the `samples/` folder, pick or create any samples, develop and test your changes over there.
+
+## Contributing Rule
+1. Push your changes to a topic branch in your fork of the repository.
+1. Ensure that your code adheres to the existing style in the sample to which
+   you are contributing.
+1. Ensure that your code has an appropriate set of unit tests which all pass.
```

Comment:
Instructions on how to run the tests locally can be helpful as most of the PR builder jobs fail.
`npm run test` on the project root can catch a lot of error before creating a PR.